runtime/drivers/duckdb/extensions.go

```diff
@@ -0,0 +1,134 @@
+package duckdb
+
+import (
+	"compress/gzip"
+	"database/sql"
+	"embed"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Since DuckDB is called from multiple packages, the extensions are installed in the init function
+func init() {
+	err := installExtensions()
+	if err != nil {
+		// If extensions cannot be installed, log the error and continue as the extensions can be downloaded
+		// Should it be fatal in order to notice the issue prior to a release?
+		log.Printf("Error preparing DuckDB extensions: %v", err)
+	}
+}
+
+//go:embed embed/extensions/*
```

Comment:
To avoid accidental commits, can we add the extension files here to `gitignore`? Might need a nested pattern like `runtime/drivers/duckdb/embed/extensions/*/*` or something like that to still maintain the `.gitkeep` file.